.github/workflows/publish.yml

```diff
@@ -60,3 +63,20 @@ jobs:
           config: "./.github/configs/cr.yaml"
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Login to GHCR
+        uses: docker/login-action@465a07811f14bebb1938fbed4728c6a1ff8901fc # v2.2.0
+        with:
+          registry: ghcr.io
+          username: ${ GITHUB_REPOSITORY_OWNER }
```

Comment:
Recommend changing to `github.actor` as it is suggested by the `login-action` [documentation](https://github.com/docker/login-action#github-container-registry)

Reply:
Done! @sabre1041 thanks for spotting this!